modules/core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryVersioning.java

```diff
@@ -610,6 +610,17 @@ public void testVersionLabelOfCopy() {
         assertEquals("0.1", version.getVersionLabel());
     }
 
+    @Test
+    public void checkVersionNotNullOnCopy() {
```

Comment:
Please start the method name with `test` instead of `check`. This makes it easier when navigating or searching to isolate test methods from others, and is the convention we have in most of our code.